errtracker/errtracker.go

```diff
@@ -66,11 +69,18 @@ func Report(snap, channel, errMsg string) (string, error) {
 
 	crashDbUrl := fmt.Sprintf("%s/%s", CrashDbURLBase, identifier)
 
+	usrBinSnapBuildID, err := osutil.GetBuildID(usrBinSnap)
```

Comment:
This is just using /usr/bin/snap -  I could include /usr/lib/snapd/snapd here too, but it seems given that they come from the same package in almost all cases this should give us the data we need(?).